apps/api/Sources/App/PairQL/NextEvansBuild/FriendPage.swift

```diff
@@ -60,6 +61,23 @@ struct FriendPage: Pair {
         var end: Int
       }
     }
+
+    struct RelatedDocument: PairNestable {
+      var title: String
+      var htmlShortTitle: String
+      var slug: String
+      var description: String
+      var isCompilation: Bool
+      var friendName: String
+      var friendSlug: String
+      var friendGender: Friend.Gender
+      var editionType: EditionType
+      var paperbackVolumes: NonEmpty<[Int]>
```

Comment:
just fyi, this is the swift NonEmpty generic type i was talking about, that statically guarantees at least one item, which i convert into that typescript type you noticed in the other PR.